### PR TITLE
Reduce deps

### PR DESCRIPTION
**Pull request by rprtr258**
throw out some dependencies

## Summary by CodeRabbit

- **New Features**
  - Introduced new functions for sequence handling and text styling in terminals.
- **Bug Fixes**
  - Simplified and enhanced test case processing and formatting for improved readability and consistency.
- **Refactor**
  - Updated import paths and refactored multiple functions across various modules for better maintainability.
- **Tests**
  - Added new test functions and refactored existing ones to align with updated functionalities.
- **Chores**
  - Changed the test command in Makefile to use a more detailed format output.

### CODE REVIEW

File: assert_test.go [hunk 1]
```diff
@@ -15,16 +16,25 @@ type User struct {
...
 }
 
+func toSlice[T any](seq fun.Seq[T]) []T {
+	var xs []T
+	seq(func(x T) bool {
+		xs = append(xs, x)
+		return true
+	})
+	return xs
+}
```
[the remark this answers is not in the document]
Comment (rprtr258) [reply assert_test.go:26]: yes, write tests to handle all possible edge cases

File: pp/printer_test.go [hunk 2]
```diff
@@ -61,354 +80,146 @@
...
+		//     [33mFie3[0m: [34;1m128[0m,
+		// }
+		// &c,
+		// -- &pp.Circular{C:(*pp.Circular)(0x6ca188)} --
+		// &pp.[32mCircular[0m{
+		//     [33mC[0m: &pp.[32mCircular[0m{...},
+		// }
+		// make(chan bool, 10),
+		// unsafe.Pointer(&regexp.Regexp{}),
+	}
```
[the remark this answers is not in the document]
Comment (rprtr258) [reply pp/printer_test.go:181]: most of function is list of testcases, the cannot be reduced
